tests/torch/test_compile_torch.py

```diff
@@ -146,15 +147,27 @@ def compile_and_test_torch_or_onnx(  # pylint: disable=too-many-locals, too-many
     else:
         inputset = (numpy.random.uniform(-100, 100, size=(n_examples, *to_tuple(input_shape))),)
 
-    # FHE vs Quantized are not done in the test anymore (see issue #177)
-    if not simulate:
+    # Compile our network with the same bitwidth in simulation and FHE
+    if qat_bits == 0:
+        n_bits_w_a = 4
+    else:
+        n_bits_w_a = qat_bits
+
+    n_bits = (
+        {
+            "model_inputs": n_bits_w_a,
+            "model_outputs": n_bits_w_a,
+            "op_inputs": n_bits_w_a,
+            "op_weights": n_bits_w_a,
+        }
+        if qat_bits == 0
+        else qat_bits
+    )
 
-        n_bits = (
-            {"model_inputs": 2, "model_outputs": 2, "op_inputs": 2, "op_weights": 2}
-            if qat_bits == 0
-            else qat_bits
-        )
+    n_rounding_bits = 6
 
+    # FHE vs Quantized are not done in the test anymore (see issue #177)
```

Comment:
Maybe you should, put the issue ID, in the right format

Reply:
We have a syntax for TODO, which link to open issues . but this is not an open one, the reference here is for reference purposes only. so this is fine as it is